Make postTweet response include errors

johnbailon opened the pull request:
Post tweet doesn't return the full response when the request is not successful. I added details from Twitter API's response to the returned object.

sivchari commented:
HI, @johnbailon 
Looks good to me. Just in case, please give me the documentation about this. Thanks.

johnbailon commented:
> HI, @johnbailon
> Looks good to me. Just in case, please give me the documentation about this. Thanks.

Sure, basically, I just added the error that would come from Twitter on to the response of PostTweet. I made sure to follow your conventions and I modified the test to take into account the changes I made.

Context: I needed to know what was causing my PostTweet requests to fail and the current version doesn't provide the erro response from Twitter.

sivchari commented:
@johnbailon 
Sorry, too late to respond.  I concern if this response is really had `errors` field. Have you confirmed it ? If yes, please paste the response. Because I want to confirm it just in case.
